tests/compile_ouroboros.py

```diff
@@ -0,0 +1,670 @@
+# -*- coding: utf-8 -*-
+"""Check modules from Ouroboros that compile with voc."""
+
+# Standard library imports
+from datetime import datetime
+from subprocess import Popen, PIPE
+import multiprocessing
+import os
+
+HERE = os.path.dirname(os.path.realpath(__file__))
+MODULES_WHITELIST = [
```

Comment:
Wouldn't it be better to do the overrides via a configuration instead of convention?
For cases like the Java implementation is buggy for a particular module or for some other reason someone wants the compiled Python version?